### PR TITLE
Introduce UI changes for ForensicArtifacts indicator type

### PR DESCRIPTION
**Pull request by tomchop**

### CODE REVIEW

File: src/components/EditObject.vue [hunk 1]
```diff
@@ -84,11 +83,10 @@ export default {
           this.isActive.value = false;
         })
         .catch(error => {
-          console.log(error);
           this.errors = error.response.data.detail
-            .filter(detail => detail.loc[1] !== "type")
+            .filter(detail => detail["loc"][2].toLowerCase() === this.typeDefinition.type)
             .map(detail => {
-              return { field: detail.loc[1], message: detail.msg };
+              return { field: detail["loc"][3], message: detail.msg };
```
Comment (udgover) [inline src/components/EditObject.vue:89]: just for my own understanding. Why are you now using 2 for type checking and returning 3 for field?

File: src/definitions/indicatorDefinitions.js [hunk 2]
```diff
@@ -2,22 +2,47 @@ export const DIAMOND_MODEL = ["adversary", "capability", "infrastructure", "vict
...
     name: "Regular expression",
     type: "regex",
     fields: [
-      { field: "created", type: "date", label: "Created", displayList: true, editable: false, width: "200px" },
+      { field: "created", type: "date", label: "Created", displayList: true, editable: false, width: "170px" },
       { field: "name", type: "text", label: "Name", displayList: true, editable: true },
       { field: "pattern", type: "code", label: "Pattern", displayList: false, editable: true },
       { field: "relevant_tags", type: "list", label: "Relevant tags", displayList: true, editable: true },
-      { field: "location", type: "text", label: "Location", displayList: true, editable: true },
+      { field: "location", type: "text", label: "Location", displayList: false, editable: true },
```
Comment (udgover) [inline src/definitions/indicatorDefinitions.js:37]: Do you think it's not worth displaying location?

**Review by udgover — COMMENTED** [review]
Two questions, otherwise LGTM.

File: src/components/EditObject.vue (hunk 1 above)
Comment (tomchop) [reply src/components/EditObject.vue:89]: FastAPI returns a weird structure that I had to guess how to parse. the [2] item is which Class the error is comming from (Regexp, ForensicArtifact) and the [3] item is which field in the Class is giving that error.

**Review by udgover — APPROVED** [review]
LGTM!

File: src/definitions/indicatorDefinitions.js (hunk 2 above)
Comment (tomchop) [reply src/definitions/indicatorDefinitions.js:37]: https://github.com/yeti-platform/yeti-feeds-frontend/issues/147 for good measure

**Pull request merged**